test/bin/manage_hypervisor_config.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reply:
I created the https://issues.redhat.com/browse/USHIFT-1565 issue to have a discussion and make this change in all the scripts if we decide it's a better way to go.